gravitee-node-cache/src/main/java/io/gravitee/node/cache/standalone/StandaloneCache.java

```diff
@@ -104,11 +104,10 @@ public Collection<V> values() {
   public V get(K key) {
     ValueWrapper<V> vw = internalCache.getIfPresent(key);
     if (vw != null) {
-      if (vw.expirationTimeMillis == 0) {
-        return vw.value;
-      } else if (System.currentTimeMillis() <= vw.expirationTimeMillis) {
-        vw.expirationTimeMillis =
-          System.currentTimeMillis() + vw.timeToLiveMillis;
```

Comment:
@kamiiiel was this really intended because it doesn't feel like the behavior we want, but there is a unit test that looks to test that 🤔

Reply:
The idea was that each time that you "get" the key, the expiration time will be extended